src/Microsoft.PowerShell.ConsoleHost/resources/ConsoleHostStrings.resx

```diff
@@ -188,4 +188,4 @@ The current session does not support debugging; execution will continue.
   <data name="SlowProfileLoadingMessage" xml:space="preserve">
     <value>Loading personal and system profiles took {0}ms.</value>
   </data>
-</root>
\ No newline at end of file
+</root>
```

Comment:
A guess: I think your script was fairly consistent with these. If it made a change in the file, it didn't add an extra newline at the end. I'm going to stop commenting on each individual one. Please make it consistent throughout if you are going to make this change.